src/tools/miropt-test-tools/src/lib.rs

```diff
@@ -98,8 +100,11 @@ pub fn files_for_miropt_test(
                 from_file = format!("{}.{}.mir", test_name, first_pass);
                 to_file = Some(second_file);
             } else {
-                let ext_re = regex::Regex::new(r#"(\.(mir|dot|html))$"#).unwrap();
-                let cap = ext_re
+                lazy_static::lazy_static! {
+                    static ref EXT_RE: Regex = Regex::new(r#"(\.(mir|dot|html))$"#).unwrap();
```

Comment:
Why do we even need a regex here? This looks like something that could be handled with ~10 lines of code via strip_suffix and a for loop?

Reply:
We already have the regex dependency for more complicated cases, so it seems silly to not use it here.

Reply:
That doesn't seem true?

```
$ rg regex src/tools/miropt-test-tools
src/tools/miropt-test-tools/Cargo.toml
7:regex = "1.0"

src/tools/miropt-test-tools/src/lib.rs
101:                let ext_re = regex::Regex::new(r#"(\.(mir|dot|html))$"#).unwrap();
```

Reply:
Ah, I was looking at tidy, not miropt-test-tools.

Reply:
`miropt-test-tools` used in `tidy` and `completest`, which both uses regex.

Reply:
Removed regex usage.